src/OpenTK.Windowing.Desktop/GameWindow.cs

```diff
@@ -247,14 +302,13 @@ public virtual void SwapBuffers()
             }
         }
 
+        /// <inheritdoc />
         public override void Close()
         {
             OnUnload(this, EventArgs.Empty);
             base.Close();
         }
 
-        #region EventHandlers
```

Comment:
Are regions against the style guide? This file is incredibly long, regions make it much easier to handle.

Reply:
Well unfortunately yes. At least the styleguides given by the stylecop.ruleset...

Reply:
That stinks. Oh well, this is fine then.